project-base/storefront/components/Pages/Order/TransportAndPayment/TransportAndPaymentContent.tsx

```diff
@@ -63,19 +65,15 @@ export const TransportAndPaymentContent: FC = () => {
         router.push(contactInformationUrl);
     };
 
-    useEffect(() => {
-        if (cart !== undefined && !cart?.items.length) {
-            router.replace(cartUrl);
-        }
-    }, [cart?.items]);
+    const canContentBeDisplayed = useOrderPagesAccess('transport-and-payment');
```

Comment:
I do not agree with it. It is not any convention we follow, nor it is something strictly followed by any of the major technologies we use. For me the point is for the code to be as readable as possible. Ideally follow natural language.

Reply:
I think we have already agreed with other FE guys that we want to mark those boolean values with this pattern, or...? 🤔 I personally don't see it any difficult to read the code I rather see it as improved because I can quickly recognize boolean value without thinking about it.

If you want to have code better readable I don't see point of naming variable as `canContentBeDisplayed` if you use directly return from `useOrderPagesAccess`. Wouldn't it be rather better readable if you in JSX know why some element shouldn't be displayed? So why not at least variable `canContentBeAccessed` or as I wrote previously something like `IsWithAccess`?

Reply:
I do not remember any such agreement, and neither has Ivan. Regarding Tom's and Michal's opinion, I was not able to get it in time for this answer. Nevertheless, the opinion that is shared by Ivan and myself, also the BE part of our team, and the industry (check react or next.js code) is to allow all prefixes such as has, can, should, are...